docs/src/components/CustomAudioMeter.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+export interface ICustomAudioMeterProps {
+    audioMonitor: any
+}
+
+export class CustomAudioMeter extends React.PureComponent<ICustomAudioMeterProps, {}> {
+    private audioMeter: HTMLDivElement;
+
+    constructor(props: ICustomAudioMeterProps) {
+        super(props);
+
+        this.refMeter = this.refMeter.bind(this);
+    }
+
+    public componentDidMount() {
```

Comment:
Shall we do UnMount to destroy the listener?

Reply:
Done